plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerNode.java

```diff
@@ -52,12 +52,10 @@
      *
      * @param workspaceId
      *         id of workspace to bind
-     * @param hostProjectsFolder
-     *         folder on the docker host where workspace should be bound
      * @throws MachineException
      *         if error occurs on binding
      */
-    void bindWorkspace(String workspaceId, String hostProjectsFolder) throws MachineException;
+    void bindWorkspace(String workspaceId) throws MachineException;
```

Comment:
There is a line on top of javadoc description which says `Project can't be bound/unbound if workspace is bound already, and vice versa.` should it still be present?